build.gradle.kts

```diff
@@ -61,39 +61,105 @@ publishing {
         maven {
             url = uri("https://repository.apache.org/service/local/staging/deploy/maven2")
         }
-
     }
 }
 
 ant.importBuild("build.xml") {old ->"ant-${old}"}
 
-
 tasks.register<Delete>("clean-default-lib")
 {
-    description = "--> clean the project libraries directory (dependencies)"
+    description = "clean the project libraries directory (dependencies)"
     delete("./build/lib")
 }
 
 tasks.register<Delete>("clean-test-lib")
 {
-    description = "--> clean the project test libraries directory (dependencies)"
+    description = "clean the project test libraries directory (dependencies)"
     delete("./build/test/lib")
 }
 
 tasks.register<Delete>("clean-build")
 {
-    description = "--> clean the project built files"
+    description = "clean the project built files"
     delete("./build")
 }
 
 tasks.register<Delete>("clean-dist")
 {
-    description = "--> clean the project dist files"
+    description = "clean the project dist files"
     delete("./dist")
 }
 
 tasks.register<Delete>("clean-runtime")
 {
-    description = "--> clean the project runtime area"
+    description = "clean the project runtime area"
     delete("./runtime")
+}
+
+tasks.register<Copy>("copy-libs")
+{
+    description = "copy the libs in lib, which are not ivy enabled"
+    from(layout.buildDirectory.dir("${project.properties["lib.dir"]}"))
+    include("**/*.jar")
+    into(layout.buildDirectory.dir("${project.properties["build.lib.dir"]}"))
+}
+
+tasks.register("compile-plugins")
+{
+    description = "compile plugins only"
+    dependsOn("init","resolve-default","deploy")
+}
+
+tasks.register<org.gradle.jvm.tasks.Jar>("jar")
+{
+    description = "make nutch.jar"
+    from(layout.buildDirectory.dir("${project.properties["conf.dir"]}/nutch-default.xml"))
+    into(layout.buildDirectory.dir("${project.properties["build.classes"]}"))
+
+    from(layout.buildDirectory.dir("${project.properties["conf.dir"]}/nutch-site.xml"))
+    into(layout.buildDirectory.dir("${project.properties["build.classes"]}"))
+}
+
+tasks.register<Copy>("runtime")
```

Comment:
This target relies upon `jar` and `job` please make sure you add the relevant `dependsUpon`... unless I am missing something.